src/changes/changes.xml

```diff
@@ -72,17 +72,21 @@ https://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd
     </release>
     -->
     <release version="0.17-SNAPSHOT" date="xxxx-yy-zz" description="Current SNAPSHOT - release to be done">
-      <action issue="RAT-377" type="add" dev="claudenw">
-        Added ability to specify the level of reporting on STANDARD files within a project.  This necessitated an addition
-        of a command line option "standard" to limit specify the level of detail in the STANDARD file reporting.  See command line
-        help for more details.  By default, there is no change in the reporting and only the presence of archives are reported.
+        <action issue="RAT-377" type="add" dev="claudenw">
```

Comment:
Github shows special characters here? Is that line breaks or tabs? (just toa void strange formatting errors when crafting the release) Thanks

Reply:
I can feel your pain - I see strange things when opening RAT in Eclipse, but will try harder :)

Reply:
In case of formatting problems the changelog will be adapted once we craft the release.